201-zerto-zca-vpn/README.md

```diff
@@ -0,0 +1,13 @@
+# Deploy Zerto Virtual Replication to Azure with a VPN
+
+This template deploys the Zerto Cloud Appliance (ZCA) to Azure. It deploys a virtual network, public ip, network interface and a ZCA virtual machine. 
+
+It also configures an [Azure VPN Gateway](https://docs.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-howto-site-to-site-resource-manager-portal) to create a site-to-site VPN connection between the Azure Virtual Network and your local on-premises network.
+
+
+1. [azuredeploy.json](./azuredeploy.json)
```

Comment:
not sure what this is for, can we just remove it?

Reply:
done